Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14-buster
+FROM node:14.15-buster
```

Comment:
this docker file is really out of date - could do with updating to match the others at some point